standalone/src/main/java/io/delta/standalone/Operation.java

```diff
@@ -0,0 +1,96 @@
+package io.delta.standalone;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * An operation that can be performed on a Delta table.
+ *
+ * An operation is tracked as the first line in delta logs, and powers `DESCRIBE HISTORY` for Delta
+ * tables.
+ */
+public final class Operation {
+
+    /**
+     * Supported operation types.
+     */
+    public enum Name {
```

Comment:
We need a `name` method to define the custom string, such `STREAMING UPDATE` or `SET TBLPROPERTIES`.

Reply:
`String getNameAsString` ? or, `String getFormattedName` ?

And your point here is that for some Name's, like `STREAMING_UPDATE`, we want the String version to be `STREAMING UPDATE` (with a space instead of an underscore).

Reply:
> And your point here is that for some Name's, like `STREAMING_UPDATE`, we want the String version to be `STREAMING UPDATE` (with a space instead of an underscore).

Since the reason we use `enum` is to unify the operation names in the transaction logs, we should be consistent with OSS Delta core.

Reply:
You can take a look at https://github.com/apache/iceberg/blob/90225d6c9413016d611e2ce5eff37db1bc1b4fc5/api/src/main/java/org/apache/iceberg/DistributionMode.java#L38 as an example.

In a second thought, we don't need to add a new method. We can just overwrite `toString` method.